pages/[id]/bridge.tsx

```diff
@@ -130,7 +144,7 @@ const Bridge = () => {
     isLoading: bridgeToL1IsLoading,
     error: bridgeToL1Error,
   } = useEasyWrite({
-    enabled: isNonZeroInput && bridgeTarget === BridgeTarget.L1 && source.chain.id === chain?.id,
```

Comment:
nit: why remove here? do we want this hook to run when the input is 0?

Reply:
Good catch,  there was a latency when testing, but now everything seems to be good.

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/157/commits/283e41df1bf91e1a6eba8e6be69699ab7fa38a85